lib/v2/gov/stats/utils.js

```diff
@@ -33,6 +33,15 @@ const parseXilan = async (item, ctx) =>
         const title = $('.xilan_titf').text();
         item.author = title.match(/来源：(.*)发布时间/)?.[1].trim() ?? '国家统计局';
         item.pubDate = timezone(parseDate(title.match(/发布时间：(.*)/)?.[1].trim() ?? item.pubDate), +8);
+
+        // Change the img href from relative href to absolute href
+        $('.xilan_con')
+            .find('img')
+            .each((_, el) => {
+                $(el).attr('src', new URL($(el).attr('src'), item.link).href);
+                // oldsrc is causing freshrss imageproxy not to work correctly
+                $(el).removeAttr('oldsrc').removeAttr('alt');
```

Comment:
Any reasons why removing `alt`?

Reply:
some 'alt' in articles have bogus addresses leaking problem, e.g
<img width="1333" alt="image" src="https://user-images.githubusercontent.com/207260/192004195-5b77b95c-1956-440d-a8e2-dfa79690c078.png">
remove it for a better article reading